Assets/Scripts/Entity/Entity.cs

```diff
@@ -43,6 +54,11 @@ public abstract class Entity : MonoBehaviour {
     public abstract void Respawn();
     public abstract void Die();
 
+    public virtual void AddBlood() {
+       // Empty because only the PlayerEntity will need this function. 
+    }
```

Comment:
Do you need this virtual function in Entity or would a normal function in PlayerEntity suffice? I couldn't find that you call `Entity.AddBlood` anywhere.